app/web/features/messages/messagelist/ControlMessageView.tsx

```diff
@@ -41,8 +42,7 @@ export default function ControlMessageView({
   );
   const { ref } = useOnVisibleEffect(onVisible);
 
-  const isCurrentUser = author?.userId === currentUserId;
-  const authorName = isCurrentUser ? "you" : firstName(author?.name);
```

Comment:
I decide to get rid of this "you" thing to simplify the i18n logic, otherwise I'd effectively have to have duplicate variants of the same message just for the current user, since we can't reliably assume we can just swap out name for "you" as in English. Not worth the pain and complexity to keep this imo.